Pod/Classes/PINRemoteImageManager.m

```diff
@@ -281,6 +283,21 @@ - (void)unlock
     [_lock unlock];
 }
 
+- (void)setValue:(nullable NSString *)value forHTTPHeaderField:(nullable NSString *)header {
+    __weak typeof(self) weakSelf = self;
+    dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^{
+        typeof(self) strongSelf = weakSelf;
+        [strongSelf lock];
+            if (value) {
+                strongSelf.httpHeaderFields[header] = value;
```

Comment:
Do you think we should change this to `strongSelf.httpHeaderFields[[header copy]] = [value copy];`? That way if it turns out someone has passed in a mutable string and changes it later we won't get in trouble?